### ISSUE TITLE
OER at University of British Columbia

### ISSUE THREAD

**Issue by dohliam** [lorrainechu3n/OOOCanadaResearchNetwork#10]
[UBC FOSS](https://github.com/ubcfoss) been trying to aggregate and keep track of OER and FOSS work happening at UBC -- not sure if you might be interested in including some of this information in your project:

* [ubc-oer](https://github.com/ubcfoss/ubc-oer) - Open Educational Resources (including projects, open textbooks, MOOCs, and open access journals
* [ubc-orgs](https://github.com/ubcfoss/ubc-orgs) - UBC Organizations that have a presence on Github (and are presumably contributing to OER/FOSS)
* [awesome-ubc-foss](https://github.com/ubcfoss/awesome-ubc-foss) - A curated list of FOSS projects at UBC

What we've discovered in this process is that there is a lot going on in terms of open research, OER, and FOSS on campus, but very little in the way of centralized information for people who want to keep up with everything. We are hoping this information helps a bit to make it easier for people who want to get involved in the community to know who's doing what.

It's great to see initiatives like this gaining momentum in schools across the country!

**Comment by char-siuu-bao** [comment]
Hey @dohliam — Thank you so much for giving this update on UBC!  You've linked some great resources in there — I will take a more thorough look when I have a bit of time tomorrow.

And yes, I was hoping to start this Github repo to address a similar problem to the one you mentioned - but at the national level. Lots of awesome work on advancing Open being done across Canada, but no centralized platform for people to provide updates. Hoping this can fill that gap!

**Comment by dohliam** [comment]
Yes there is definitely a need for something like this, to encourage collaboration in the spirit of open source/open access and help avoid working in silos as happens all too often. Keep up the good work! :+1:
